comptoir-ejb/src/main/java/be/valuya/comptoir/model/commercial/ItemVariantSale.java

```diff
@@ -48,6 +42,9 @@
     @NotNull
     @Nonnull
     private Stock stock;
+    @Column(name = "FORCE_CUSTOMER_LOYALTY")
+    @CheckForNull
+    private Boolean forceCustomerLoyalty;
```

Comment:
Thinking of it, this should be a non-nullable 'useForCustomerLoyalty' field, set by the backend once the itemSale is created.